internal/manifests/targetallocator/deployment_test.go

```diff
@@ -61,7 +61,7 @@ func TestDeploymentNewDefault(t *testing.T) {
 	}
 
 	// test
-	d := Deployment(params)
+	d, _ := Deployment(params)
```

Comment:
suggestion: let's check the error here too... depending on the conversation above we may actually just remove this.

Reply:
bump on this